### DISCUSSION TITLE
UV-Vis data

### DISCUSSION THREAD

**Discussion by zxdawn** [radis/radis#645] [Q&A] [answered]
Hi, thanks for the nice package! May I ask if I wanna get absorption coefficient for UV-Vis band, where should I go?

**Comment by dcmvdbekerom** [comment, marked as answer]
For UV-vis cross sections I suggest to look here  https://www.uv-vis-spectral-atlas-mainz.org/uvvis/

UV vis linelists for line-by-line modelling (as in radis) are not very common to my knowledge, perhaps exomol had some? 

For visible there's also specair or lifbase you could consider depending on the molecule you're interested in.

**Reply by zxdawn** [reply]
Thanks! I'm interested in NO2. It seems the first link has it: https://uv-vis-spectral-atlas-mainz.org/uvvis/cross_sections/Nitrogen%20oxides/NO2.spc

BTW, how did you calculate absorption cross-section for different pressures and temperatures (e.g. different atmospheric layers)? I see the dataset usually provides one specific value.
